README.md

```diff
@@ -12,15 +14,37 @@ The input to the pipeline is a standard sample sheet (passed as `--input samples
 | ------- | ----------------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- | ---------- |
 | SampleA | sampleA.mlst.json | meta1      | meta2      | meta3      | meta4      | meta5      | meta6      | meta7      | meta8      |
 
-The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Validation of the sample sheet is performed by [nf-validation](https://nextflow-io.github.io/nf-validation/).
+The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Validation of the sample sheet is performed by [nf-validation](https://nextflow-io.github.io/nf-validation/). Details on the columns can be found in the [Full samplesheet](docs/usage.md#full-samplesheet) documentation.
 
 # Parameters
 
 The main parameters are `--input` as defined above and `--output` for specifying the output results directory. You may wish to provide `-profile singularity` to specify the use of singularity containers and `-r [branch]` to specify which GitHub branch you would like to run.
 
 ## Metadata
 
-In order to customize metadata headers, the parameters `--metadata_1_header` through `--metadata_8_header` may be specified.
+In order to customize metadata headers, the parameters `--metadata_1_header` through `--metadata_8_header` may be specified. These parameters are used to re-name the headers in the final metadata table from the defaults (e.g., rename `metadata_1` to `country`).
+
+## Profile dists
+
+The following can be used to adjust parameters for the [profile_dists][] tool.
```

Comment:
I thought this was a mistake, but I didn't know you could use markdown like this to link to another repo. Interesting,

Reply:
Agreed - good to know :)

Reply:
Yes. Or, specifically `[profile_dists][]` is shorthand for `[profile_dists][profile_dists]` where the second `[profile_dists]` refers to the references at the bottom of this file.